packages/formation/sass/modules/_m-button.scss

```diff
@@ -125,37 +122,52 @@ button.short {
 
 // Opens / closes a sidebar navigation on detail pages.
 .va-btn-sidebarnav-trigger {
-  background: $color-gray-lightest;
-  border-radius: 0;
-  display: block;
-  color: $color-link-default;
-  margin: 1rem 0 2rem;
-  text-align: left;
+  margin: 0;
+  padding: 0;
 
-  span {
-    align-items: center;
-    display: flex;
+  &.fixed-trigger {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: $top-layer - 1;
   }
 
-  b {
-    flex: 1 1 10rem
+  .button-wrapper {
+    position: absolute;
+    width: 100%;
+    padding: units(1) units(1) units(2);
   }
 
-  svg {
-    display: inline-block;
-    height: 1rem;
-    width: 1rem;
-    fill: $color-link-default;
-  }
+  button {
+    background-color: $color-white;
+    color: $color-primary;
+    margin: 0;
+    padding: units(1.5);
+    border: 1px solid $color-primary;
+    border-radius: 4px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
This is 100% something I should know but don't: Is this duplicating any existing styling we've got? I'd love to trim down our styles if we can.

Reply:
We have styles similar to this used in https://github.com/department-of-veterans-affairs/vets-website/blob/4b679fddefe2bc1d34e2f14eb9ef171c46d6bffe/src/applications/personalization/profile/sass/profile-mobile-subnav.scss, but the markup for that menu is different/more complex than this simple button. In the file I linked to, the button looks identical to this button, but functions entirely differently so the markup (and thus CSS) is different.

Reply:
Also, styles similar to this (for the Pittsburgh VA's menu) live outside of formation in https://github.com/department-of-veterans-affairs/vets-website/blob/cbfe8334fd5a2d2db7bf635b916cfa808a9670fc/src/platform/site-wide/sass/modules/_m-side-nav.scss